packages/postgres/config/dbCollections.ts

```diff
@@ -1,4 +1,4 @@
-export const messagesTableName = 'storequeuedMessage'
+export const messagesTableName = 'storequeuedmessage'
```

Comment:
What's the reason behind this naming 'storeXXX'? is it a convention or something?
To me it would be more natural to simply call the table under the name of the entity it is representing, such as 'queuedmessage' and 'livesession'

Reply:
The main reason is that I wanted to keep the same names as the schemas in the WebSocket MPR server. 

Solved! The names are now changed to the proposed ones.

Reply:
Yes, but... I don't understand why those names were used in the WebSocket MPR server :D